.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release artifact
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Would be good to verify that dynver works correctly and publishes a snapshot version to Snapshots Sonatype.

In my projects I use https://github.com/olafurpg/sbt-ci-release which is designed for this flow, but I don't want to mess with this part of the build just yet